test/integration/cli/help.bats

```diff
@@ -5,126 +5,126 @@ load ${BASE_TEST_DIR}/helpers.bash
 @test "cli: show info" {
   run machine
   [ "$status" -eq 0  ]
-  [[ ${lines[0]} =~ "Usage:"  ]]
-  [[ ${lines[1]} =~ "Create and manage machines running Docker"  ]]
+  [[ ${lines[0]} =~ "Usage:"  ]] || false
```

Comment:
I'm not sure what problem this is solving, but I'm fairly positive it'll break the test; AFAIU, `bats` relies on `set -e`, for which the Bash manual has this to say (https://www.gnu.org/software/bash/manual/bashref.html#index-set):
> Exit immediately if a pipeline (see Pipelines), which may consist of a single simple command (see Simple Commands), a list (see Lists), or a compound command (see Compound Commands) returns a non-zero status. The shell does not exit if the command that fails is part of the command list immediately following a while or until keyword, part of the test in an if statement, part of any command executed in a && or || list except the command following the final && or ||, any command in a pipeline but the last, or if the command’s return status is being inverted with !. If a compound command other than a subshell returns a non-zero status because a command failed while -e was being ignored, the shell does not exit. A trap on ERR, if set, is executed before the shell exits.

What this means in this context is that the `|| false` makes this expression explicitly always "true" as far as `set -e` is concerned (it won't stop the script, or force the test to fail).